teerex/sgx-verify/Cargo.toml

```diff
@@ -15,6 +15,7 @@ workspace = true
 base64 = { workspace = true }
 chrono = { workspace = true }
 der = { workspace = true }
+getrandom = { workspace = true }
```

Comment:
I think this is the problem. I see that you added that line in order to fix the wasm build issue in the parachain. However, I think that the problem is that an std-leak of sorts resulted in the compiler error requesting the `js` feature.